lib/timeutils/unixtime.h

```diff
@@ -69,6 +69,37 @@ unix_time_is_timezone_set(const UnixTime *self)
   return self->ut_gmtoff != -1;
 }
 
+static inline
+void dump_unix_time(const UnixTime *ut, GString *output)
```

Comment:
can you move the return value type to the previous line? that's the convention we are using, e.g.

```
static inline void
dump_unix_time(...)
```

Reply:
also, we might hide this in the .c file, I don't see why this should be inline.